lxd/patches.go

```diff
@@ -986,7 +1021,15 @@ func upgradeFromStorageTypeZfs(name string, d *Daemon, defaultPoolName string, d
 			poolConfig["source"] = shared.VarPath("disks", defaultPoolName+".img")
 			err := os.Rename(oldLoopFilePath, poolConfig["source"])
 			if err != nil {
-				return err
+				output, err := storageRsyncCopy(oldLoopFilePath, poolConfig["source"])
```

Comment:
shared.FileMove will only move files, not copy directories including contents, no?